MovieRate/src/components/Search/Search.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, useRef } from 'react';
+import MovieBar from '../MovieBar/MovieBar';
+import './SearchStyle.css';
+
+function Search() {
+  const [state, setState] = useState({
+    searchedMovies: [],
+    searchText: ''
+  });
+
+  const ref = useRef();
+  const { searchedMovies, searchText } = state;
+  const url = `https://api.themoviedb.org/3/search/movie?query=${searchText}&api_key=6fea6800ad53c60d927efe6f58809c01`;
+
+  useEffect(() => {
+    fetch(url)
```

Comment:
I'd suggest to move fetch to a separate function in separate file called for example `api.js` and keep all the functions for data fetching/posting/updating in that file.